.github/ISSUE_TEMPLATE/feature-request.md

```diff
@@ -0,0 +1,27 @@
+---
+name: Request a Feature
+about: "Template for noting future planned features. Please ask for approval in the Discord if you aren't an organization Member before posting a feature request"
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+<!--
+This is a loose format, write this however you need to
+Remove these comments before submitting
+-->
+
+# Description
+<!--
+Explain your request in as much detail as you can
+-->
+
+I want Goldfish crackers
+
+# Media
+<!--
+If applicable, add images or even a video to help explain your suggestion
+-->
+
+![Example Screenshot](https://example.com/thisimageisntreal.png)
```

Comment:
What do you think about this instead? Deeper layout, asks for more and provides more. Higher bar for entry as well when people see this.

```suggestion
---
---
name: Request a Feature
about: "Template for noting future planned features. Ask for approval in the Discord if you aren't an organization Member before posting a feature request to get quick feedback on it, and find out if it's something where generally interested in."
title: ''
labels: ''
assignees: ''

---

<!--
  This is a loose format and does not strictly need to be kept, adjust as you'd like.
  Ensuere you remove all comments (anything after <!-- to the ending characters, including the start and end characters in question) before submitting your issue.
-->

# Identified Issue or Content Gap
<!--
  Explain what you have identified as an issue or gap, be it a power-imbalence, something you feel would make the game better, or simply a cool idea you've come up with.
-->
I am currently hungry. Note that I am not *always* hungry.  
This results in me being cranky, and not wanting to do anything.

# Proposed Solution  
<!--
  Explain your idea or fix in as much detail as you can.
-->
I feel that eating gold fish crackers would solve this, as they would fill the vacuous hole within myself.

# Implementation Details  
<!--
  If capable, adding details on how you would implement this suggestion is greatly appreciated.
  If you are currently unaware, or lack the required skills to understand how to implement it, leave this feild blank. Someone else may fill it in later.
-->
Goldfish crackers can be obtained from the kitchen, my mom put them there last weekend.  
About 47 would fill me full. Below is the sequence of events to get them, in psuedocode.
\```
[0] MOVE_TO 'kitchen'
[1] OPEN 'cupboard'
[2] GET 'gf_crackers'
!MARK 'yum'
[3] GET 'goldfish'
[4] EAT 'goldfish'
!IF 'gf_crackers' != 0
  GOTO 'yum'
[5] SMILE_BACK
\```

# Media  
<!--
  If applicable, add images or even a video to help explain your suggestion.
  You can simply drag image files onto this page and GitHub will embed them for you.
  This can be an example of the problem, or a prototype/example of the suggestion.
-->
![Example Screenshot](https://picsum.photos/512/288)
```
Had to escape the code block :P